internal/controller/streams_helper.go

```diff
@@ -89,7 +89,7 @@ func streamMutationObjectData(sm *streamManager, metadata *pb.KubernetesObjectDa
 }
 
 // sendClusterMetadata sends a message to indicate current cluster metadata
-func sendClusterMetadata(ctx context.Context, sm *streamManager) error {
+func sendClusterMetadata(ctx context.Context, sm *streamManager, cniStatus pb.CniPluginStatus) error {
```

Comment:
```suggestion
func sendClusterMetadata(ctx context.Context, sm *streamManager) error {
```
based on comments above